### PR TITLE
Update Supported environments for 0.47.0 release

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
runtimes/idteam/issues/1434

Update Supported environments for 0.47.0 release.

Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/openj9_support.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ columns will be removed over time.
...
+| 0.48.0         | Nov 2024 (1)   |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>   |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |    :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>       |     :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>              |    :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>             |
```
Comment (pshipton) [inline docs/openj9_support.md:53]: 0.48 doesn't support Java 22.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/openj9_support.md (hunk 1 above)
Comment (Sreekala-Gopakumar) [reply docs/openj9_support.md]: Updated.

**Review by pshipton — APPROVED** [review]

**Pull request merged**